[2023-03-22T04:05:01.144+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: CarFine.get_file scheduled__2023-03-22T04:00:00+00:00 [queued]>
[2023-03-22T04:05:01.151+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: CarFine.get_file scheduled__2023-03-22T04:00:00+00:00 [queued]>
[2023-03-22T04:05:01.151+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T04:05:01.151+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 1
[2023-03-22T04:05:01.151+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-22T04:05:01.160+0000] {taskinstance.py:1303} INFO - Executing <Task(BashOperator): get_file> on 2023-03-22 04:00:00+00:00
[2023-03-22T04:05:01.164+0000] {standard_task_runner.py:55} INFO - Started process 765 to run task
[2023-03-22T04:05:01.165+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'CarFine', 'get_file', 'scheduled__2023-03-22T04:00:00+00:00', '--job-id', '268', '--raw', '--subdir', 'DAGS_FOLDER/CarFineDAG.py', '--cfg-path', '/tmp/tmp49unfu5a']
[2023-03-22T04:05:01.165+0000] {standard_task_runner.py:83} INFO - Job 268: Subtask get_file
[2023-03-22T04:05:01.203+0000] {task_command.py:388} INFO - Running <TaskInstance: CarFine.get_file scheduled__2023-03-22T04:00:00+00:00 [running]> on host f331b8a34e72
[2023-03-22T04:05:01.250+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=CarFine
AIRFLOW_CTX_TASK_ID=get_file
AIRFLOW_CTX_EXECUTION_DATE=2023-03-22T04:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-03-22T04:00:00+00:00
[2023-03-22T04:05:01.251+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2023-03-22T04:05:01.251+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'wget https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json ']
[2023-03-22T04:05:01.258+0000] {subprocess.py:86} INFO - Output:
[2023-03-22T04:05:01.261+0000] {subprocess.py:93} INFO - --2023-03-22 04:05:01--  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:05:01.267+0000] {subprocess.py:93} INFO - Resolving www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)... 162.125.81.15, 2620:100:6035:15::a27d:550f
[2023-03-22T04:07:11.965+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:07:11.966+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:07:11.966+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:07:11.966+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:07:12.966+0000] {subprocess.py:93} INFO - --2023-03-22 04:07:12--  (try: 2)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:09:23.034+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:09:23.034+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:09:23.035+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:09:23.035+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:09:25.034+0000] {subprocess.py:93} INFO - --2023-03-22 04:09:25--  (try: 3)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:11:36.154+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:11:36.155+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:11:36.155+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:11:36.156+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:11:39.154+0000] {subprocess.py:93} INFO - --2023-03-22 04:11:39--  (try: 4)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:13:49.274+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:13:49.274+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:13:49.275+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:13:49.275+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:13:53.274+0000] {subprocess.py:93} INFO - --2023-03-22 04:13:53--  (try: 5)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:16:04.442+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:16:04.442+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:16:04.442+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:16:04.442+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:16:09.442+0000] {subprocess.py:93} INFO - --2023-03-22 04:16:09--  (try: 6)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:18:19.609+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... failed: Connection timed out.
[2023-03-22T04:18:19.610+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|2620:100:6035:15::a27d:550f|:443... failed: Cannot assign requested address.
[2023-03-22T04:18:19.610+0000] {subprocess.py:93} INFO - Retrying.
[2023-03-22T04:18:19.610+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:18:25.610+0000] {subprocess.py:93} INFO - --2023-03-22 04:18:25--  (try: 7)  https://www.dl.dropboxusercontent.com/s/q1de0kkhhxp5vo9/data.json
[2023-03-22T04:18:25.694+0000] {subprocess.py:93} INFO - Connecting to www.dl.dropboxusercontent.com (www.dl.dropboxusercontent.com)|162.125.81.15|:443... connected.
[2023-03-22T04:18:26.965+0000] {subprocess.py:93} INFO - HTTP request sent, awaiting response... 200 OK
[2023-03-22T04:18:26.966+0000] {subprocess.py:93} INFO - Length: 259778 (254K) [text/plain]
[2023-03-22T04:18:26.967+0000] {subprocess.py:93} INFO - Saving to: ‘data.json’
[2023-03-22T04:18:26.967+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:18:27.163+0000] {subprocess.py:93} INFO -      0K .......... .......... .......... .......... .......... 19%  254K 1s
[2023-03-22T04:18:27.348+0000] {subprocess.py:93} INFO -     50K .......... .......... .......... .......... .......... 39%  270K 1s
[2023-03-22T04:18:27.488+0000] {subprocess.py:93} INFO -    100K .......... .......... .......... .......... .......... 59%  357K 0s
[2023-03-22T04:18:27.632+0000] {subprocess.py:93} INFO -    150K .......... .......... .......... .......... .......... 78%  347K 0s
[2023-03-22T04:18:27.696+0000] {subprocess.py:93} INFO -    200K .......... .......... .......... .......... .......... 98%  787K 0s
[2023-03-22T04:18:27.696+0000] {subprocess.py:93} INFO -    250K ...                                                   100% 6.87T=0.7s
[2023-03-22T04:18:27.697+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:18:27.697+0000] {subprocess.py:93} INFO - 2023-03-22 04:18:27 (347 KB/s) - ‘data.json’ saved [259778/259778]
[2023-03-22T04:18:27.698+0000] {subprocess.py:93} INFO - 
[2023-03-22T04:18:27.699+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-03-22T04:18:27.749+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=CarFine, task_id=get_file, execution_date=20230322T040000, start_date=20230322T040501, end_date=20230322T041827
[2023-03-22T04:18:27.794+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-22T04:18:27.813+0000] {taskinstance.py:2585} INFO - 1 downstream tasks scheduled from follow-on schedule check
